main.go

```diff
@@ -683,11 +683,30 @@ func main() {
 
 	log.Printf("* xcodebuild_version: %s (%s)", xcodebuildVersion.Version, xcodebuildVersion.BuildVersion)
 
-	xcprettyVersion, err := cmd.GetXcprettyVersion()
-	if err != nil {
-		log.Warnf("Failed to get the xcpretty version! Error: %s", err)
-	} else {
-		log.Printf("* xcpretty_version: %s", xcprettyVersion)
+	// Detect xcpretty version
+	if configs.OutputTool == "xcpretty" {
+		fmt.Println()
+		log.Infof("Checking output tool")
+		installed, err := IsXcprettyInstalled()
+		if err != nil {
+			fail("Failed to check if xcpretty is installed, error: %s", err)
```

Comment:
do not fail if xcpretty is not installed, the step is supposed to run test. in case of issue with xcpretty, simple log warning message and set the output tool to xcodebuild